recorder.js

```diff
@@ -22,6 +24,7 @@ function parse_game_stats(){
                 }
                 if(score != ""){
                     game['scores'].push({name, score, kills, deaths});
+                    teams[i].push(name);
```

Comment:
What happens to the team names when a user leaves or joins mid-match? Is this something worth accounting for?

Reply:
I still need to do more testing for this- so far the answer is "whoever is in at the end of the game is who counts."

Reply:
Sounds like a reasonable place to start.